examples/browser-webrtc/src/main.rs

```diff
@@ -0,0 +1,145 @@
+use anyhow::Result;
+use axum::extract::{Path, State};
+use axum::http::header::CONTENT_TYPE;
+use axum::http::StatusCode;
+use axum::response::{Html, IntoResponse};
+use axum::{http::Method, routing::get, Router};
+use futures::StreamExt;
+use libp2p::{
+    core::muxing::StreamMuxerBox,
+    core::Transport,
+    identity,
+    multiaddr::{Multiaddr, Protocol},
+    ping,
+    swarm::{keep_alive, NetworkBehaviour, SwarmBuilder, SwarmEvent},
+};
+use libp2p_webrtc as webrtc;
+use rand::thread_rng;
+use std::{
+    net::Ipv6Addr,
+    net::{IpAddr, Ipv4Addr, SocketAddr},
+};
+use tower_http::cors::{Any, CorsLayer};
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    env_logger::builder()
+        .parse_filters("browser_webrtc_example=debug,libp2p_webrtc=info,libp2p_ping=debug")
+        .parse_default_env()
+        .init();
+
+    let id_keys = identity::Keypair::generate_ed25519();
+    let local_peer_id = id_keys.public().to_peer_id();
+    let transport = webrtc::tokio::Transport::new(
+        id_keys,
+        webrtc::tokio::Certificate::generate(&mut thread_rng())?,
+    )
+    .map(|(peer_id, conn), _| (peer_id, StreamMuxerBox::new(conn)))
+    .boxed();
+
+    let behaviour = Behaviour {
+        ping: ping::Behaviour::new(ping::Config::new()),
+        keep_alive: keep_alive::Behaviour,
+    };
+
+    let mut swarm = SwarmBuilder::with_tokio_executor(transport, behaviour, local_peer_id).build();
+
+    let address_webrtc = Multiaddr::from(Ipv6Addr::LOCALHOST)
+        .with(Protocol::Udp(0))
+        .with(Protocol::WebRTCDirect);
+
+    swarm.listen_on(address_webrtc.clone())?;
+
+    let address = loop {
+        if let SwarmEvent::NewListenAddr { address, .. } = swarm.select_next_some().await {
+            log::info!("Listening on: {address}");
+
+            break address;
+        }
+    };
+
+    let addr = address.with(Protocol::P2p(*swarm.local_peer_id()));
+
+    // Serve the multiaddress over HTTP
```

Comment:
```suggestion
    // Serve WASM blob and server multiaddress over HTTP
```

More accurate, right?

Reply:
Indeed, it serves many masters. Added.

Reply:
:D